src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/_shipment.html.twig

```diff
@@ -27,7 +27,11 @@
             <p>{{ shipment.tracking }}</p>
         </div>
     {% endif %}
-
+    {% if shipment.state == shipped %}
```

Comment:
Should this button be available for shipments that are ready to be shipped? IMHO, yes, it should. /cc @CoderMaggie 

```suggestion
    {% if shipment.state is not cart %}
```

Or we can even consider usage of const, but that would be precedent.